ValveKeyValue/ValveKeyValue.Test/Test Data/Text/kv_with_base_only.vdf

```diff
@@ -0,0 +1 @@
+﻿#base "file.vdf"
```

Comment:
To prevent confusion:
```suggestion
﻿#base "this_file_does_not_exist.vdf"
```

Reply:
Ah I forgot to explicitly test non-existent `#base` and `#include` files. I've updated some of the existing tests to do so. Is my change adequate?